android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -1715,7 +1716,7 @@ static public BraveActivity getBraveActivity() {
             return (BraveActivity)ref;
         }
 
-        return null;
+        throw new ActivityNotFoundException("BraveActivity Not Found");
```

Comment:
The idea was to mark function `throws ActivityNotFoundException` to enforce handling this case everywhere we call it.

Reply:
as @samartnik mentioned we need to catch `ActivityNotFoundException` everywhere we use getBraveActivity.

Reply:
updated